packages/webpack/test/scaffold.js

```diff
@@ -116,12 +116,18 @@ const defaultGlobals = () => ({
   URL,
 })
 
+/**
+ * @typedef {object} RunReturn
+ * @property {import('vm').Context} context - VM context object
+ * @property {any} result - Return value from the executed code
+ */
```

Comment:
(suggestion)

```suggestion

/** @import {Context} from 'node:vm' */

/**
 * @typedef {object} RunReturn
 * @property {Context} context - VM context object
 * @property {any} result - Return value from the executed code
 */
```

Reply:
I intended the inline import as an indication that it's the only place it's being used.